docs/eog/case-studies-cw20-timeline.rst

```diff
@@ -0,0 +1,145 @@
+--------------------
+
+Month -1 (March 2020)
+^^^^^^^^^^^^^^^^^^^^^^
+
+Agenda
+  - We reorganised agenda for online format by
+   
+    - Removing in-person social programme (walk in Belfast Botanical Gardens, visit to Ulster Museum, Drinks Reception, Conference Dinner).
+    - Adding breaks and buffer time.
+    - Adjusting timings of sessions and rearranging order. 
+      
+  - Confirmed lightning talk schedule.
+  - Made the final agenda available.
+   
+Communication
+  - We closed registration and contacted registrants through Eventbrite as soon as the decision was made that the event would be moving online to notify them, and included information on
+   
+    - The full refund policy if they were no longer able to attend the event (the deadline for a full refund was 7 days before the event start date).
+    - The reduced cost of tickets and how they would be refunded if they still wanted to attend (partial refunds for the difference would be issued during the week before the event, after the deadline for full refunds passed).
+    - Guidance on how to claim back travel and accommodation costs.
+      
+  - We had discussions with the venue and accommodation about postponing the in-person arrangements to a future date.
+   
+    - We were able to rearrange everything (venue, catering, accommodation) except for one of the hotels who were unable to refund/reschedule our reservation under the circumstances.      
+      
+  - Followed up with sponsors and offered to renegotiate packages.
+  - Followed up with speakers and workshop facilitators, confirming any changes to their sessions, and provided office hours to test their setup and answer any questions.
+  - Contacted the event delivery team with new duties roster, held pre-event meetings to test the infrastructure and make sure everyone was comfortable with their role. 
+  - In the days before the event, we sent emails to registrants through Eventbrite reminding them to review the Code of Conduct, instructions to install and test the Zoom software, and information about how to prepare for certain sessions. 
+   
+    - We also stressed that CW20 would be as flexible on the day as possible, as many people were adjusting to a new way of working, caring responsibilities during the day, unstable internet, and to the associated emotional pressures. 
+    - We provided guidance for everyone to be kind, understanding and flexible. 
+      
+  - Links to the relevant Zoom room and note-taking document were sent via email through Eventbrite and Slack at the start of each day.
+   
+Resources
+  - We set up a uCONFLY instance for management of Discussion Group Reports, Mini-workship and Demo session notes, Collaborative Ideas and Hack Day ideas which also involved
+   
+    - Creating Google Doc templates for each of these document types.
+    - Generating a number of each document type for dissemination during CW20 so that participants would not be responsible for yet another platform.
```

Comment:
```suggestion
    - Generating a number of each document type for dissemination during CW20 so that participants would not have to login and use yet another platform.
```